src/ServiceControl.Audit/Infrastructure/Settings/LoggingSettings.cs

```diff
@@ -36,47 +29,26 @@ LogLevel InitializeLevel(string key, LogLevel defaultLevel)
             }
             catch
             {
-                InternalLogger.Warn($"Failed to parse {key} setting. Defaulting to {defaultLevel.Name}.");
+                InternalLogger.Warn($"Failed to parse {logLevelKey} setting. Defaulting to {defaultLevel.Name}.");
                 return defaultLevel;
             }
         }
 
         // SC installer always populates LogPath in app.config on installation/change/upgrade so this will only be used when
         // debugging or if the entry is removed manually. In those circumstances default to the folder containing the exe
-        static string DefaultLogLocation()
-        {
-            var assemblyLocation = Assembly.GetExecutingAssembly().Location;
-            return Path.Combine(Path.GetDirectoryName(assemblyLocation), ".logs");
-        }
+        static string DefaultLogLocation() => Path.Combine(AppContext.BaseDirectory, ".logs");
 
-        public Microsoft.Extensions.Logging.LogLevel ToHostLogLevel()
+        public Microsoft.Extensions.Logging.LogLevel ToHostLogLevel() => LogLevel switch
         {
-            if (LoggingLevel == LogLevel.Debug)
-            {
-                return Microsoft.Extensions.Logging.LogLevel.Debug;
-            }
-            if (LoggingLevel == LogLevel.Error)
-            {
-                return Microsoft.Extensions.Logging.LogLevel.Error;
-            }
-            if (LoggingLevel == LogLevel.Fatal)
-            {
-                return Microsoft.Extensions.Logging.LogLevel.Critical;
-            }
-            if (LoggingLevel == LogLevel.Warn)
-            {
-                return Microsoft.Extensions.Logging.LogLevel.Warning;
-            }
-            if (LoggingLevel == LogLevel.Info)
-            {
-                return Microsoft.Extensions.Logging.LogLevel.Information;
-            }
-            if (LoggingLevel == LogLevel.Trace)
-            {
-                return Microsoft.Extensions.Logging.LogLevel.Trace;
-            }
-
-            return Microsoft.Extensions.Logging.LogLevel.None;
-        }
+            _ when LogLevel == LogLevel.Trace => Microsoft.Extensions.Logging.LogLevel.Trace,
+            _ when LogLevel == LogLevel.Debug => Microsoft.Extensions.Logging.LogLevel.Debug,
+            _ when LogLevel == LogLevel.Info => Microsoft.Extensions.Logging.LogLevel.Information,
+            _ when LogLevel == LogLevel.Warn => Microsoft.Extensions.Logging.LogLevel.Warning,
+            _ when LogLevel == LogLevel.Error => Microsoft.Extensions.Logging.LogLevel.Error,
+            _ when LogLevel == LogLevel.Fatal => Microsoft.Extensions.Logging.LogLevel.Critical,
+            _ => Microsoft.Extensions.Logging.LogLevel.None
+        };
+
+        const string logLevelKey = "LogLevel";
```

Comment:
> Use PascalCase for constant names, both fields and local constants.
https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names

That would be my preference but I realize we have no enforcement so it is just my opinion